python/lsst/sims/utils/coordinateTransformations.py

```diff
@@ -103,7 +62,194 @@ def calcLmstLast(mjd, longRad):
     last %= 24.
     return lmst, last
 
-def raDecToAltAzPa(raRad, decRad, longRad, latRad, mjd):
+
+def horizontalFromEquatorial(ra, dec, mjd, longitude, latitude):
```

Comment:
I'm not sure if there needs to be or not, but why isn't there an equatorialFromHorizontal method?